Mikhail_Khudyakov/src/components/Header/Header.jsx

```diff
@@ -1,57 +1,22 @@
-import 'bootstrap/dist/css/bootstrap.min.css';
 import React, {Component} from 'react';
-import {
-    Collapse,
-    Navbar,
-    Container,
-    NavbarToggler,
-    NavbarBrand,
-    Nav,
-    NavItem,
-    Button,
-    NavLink
-} from 'reactstrap';
-
-import Login from './Login.jsx';
+import {Navbar, Container, NavbarBrand, Nav, NavItem, NavLink,} from 'reactstrap';
```

Comment:
```suggestion
import {Navbar, Container, NavbarBrand, Nav, NavItem, NavLink} from 'reactstrap';
```
Когда в строчку - trailing comma не нужна